website/docs/docs/dbt-cloud/dbt-cloud-enterprise/audit-log.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Using the Audit Log for dbt Cloud Enterprise"
+id: audit-log
+description: "You can troubleshoot possible issues and provide security audits by reviewing event activity in your organization."
+sidebar_label: "Using the Audit log"
+---
+
+To review actions performed by people in your organization, dbt provides logs of audited user and system events. The dbt Cloud Audit log lists events triggered in your organization within the last 90 days.
+
+Use the Audit log to quickly review the actions performed by members of your organization. The Audit log includes details such as who performed the action, what the action was, and when it was performed. You can use these details to troubleshoot access issues, perform security audits, or analyze specific events. You must be an Org Owner to access the Audit log.
+
+## Accessing the Audit log
+
+To access Audit Log, click **Account Settings** and **Audit Log**.
+
+<div style={{maxWidth: '150px'}}>
```

Comment:
@JKarlavige I added spacing and the images appear large still. Any ideas?

Reply:
@runleonarun This should now be fixed. It looks like the .docImage class which wraps the image had a width of 600px. I added a max-width for this class to prevent the image from being wider than the div wrapping the lightbox.

Reply:
Thank you Jason! They are way more reasonably-sized now ☺️